src/rulesets/oas/index.ts

```diff
@@ -87,7 +88,7 @@ export const commonOasRules = (): IRuleDeclaration => ({
     type: RuleType.VALIDATION,
     summary: 'Path parameters are correct and valid.',
     enabled: true,
-    severity: RuleSeverity.ERROR,
+    severity: ValidationSeverity.Error,
```

Comment:
using `ValidationSeverity` is a little weird since the enums resolve to numbers. So really this is like saying `severity: 50` (which doesn't make too much sense to me), maybe we should update `IRuleDeclaration` to have `severity: ValidationSeverityLabel`?

Reply:
Numeric severity values are more useful from a programmatic perspective.

```ts
if (validation.severity >= RuleSeverity.ERROR) {
    // sound the alarm!
}
```

Human facing stuff should now use `severityLabel`.